pyxem/components/reduced_intensity_correction_component.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2019 The pyXem developers
+#
+# This file is part of pyXem.
+#
+# pyXem is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# pyXem is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with pyXem.  If not, see <http://www.gnu.org/licenses/>.
+"""
+A HyperSpy fitting component to fit a 4th order polynomial that goes to zero
+intensity at zero scattering to the reduced intensity profile. This is done to
+reduce the effects of multiple scattering in the pdf.
+
+"""
+import numpy as np
+
+from hyperspy.component import Component
+
+
+class ReducedIntensityCorrectionComponent(Component):
```

Comment:
Why are you not using the hyperspy existing Expression or Polynomial component for whatever you use this for?

Reply:
Good point. I planned to change this component so as to restrict it to also be zero at your desired s_max. I will add that as an option by next week, and hence it will need a specific component.

Reply:
Are you 100% sure you can't do that with existing hyperspy components too?